src/optimus/optimizations/minify.clj

```diff
@@ -23,10 +24,14 @@
       (str/replace "\r\n" "\n")
       (str/replace "\r" "\n")))
 
+(def ^String babel
+  (slurp (clojure.java.io/resource "babel.js")))
```

Comment:
Nitpick: Use the `io/` alias here. Feel free to also fix the old uglify.js-def as well. :)